### DISCUSSION TITLE
How to clear or reset filter?

### DISCUSSION THREAD

**Discussion by ntbit** [revolist/revogrid#611] [General]
How to clear or reset filter on specific colum 

![image](https://github.com/user-attachments/assets/cadeb754-d0a5-41d1-9db1-434eb57ae9b5)

**Comment by revolist** [comment]
Hi @ntbit,

To get a quicker response, it would be helpful if you could provide a small CodeSandbox example. This way, I can better understand your specific use case regarding filtering.

I acknowledge that filtering isn’t extensively covered in the documentation. However, you can find an example of how to clear or reset the filter on a specific column here: [Filter Example](https://codesandbox.io/p/sandbox/rg-filtering-defined-z9mzgr?file=%2Fpackage.json&workspaceId=b937c3e5-f7f0-45a6-aa08-b9f120dcd9b0).

Also, please ensure you’re using at least version 4.10.24 for optimal functionality.

Another option would be to access filter plugin directly with the help of [getPlugins](https://rv-grid.com/guide/api/revoGrid.html#getPlugins-Promise-PluginBaseComponent-) method

Let me know if you have any further questions!

Best,
Revolist team

**Reply by ntbit** [reply]
That's exactly what I am looking for, Thank you so much.
